WordPress/UITests/Tests/NotificationTests.swift

```diff
@@ -13,9 +13,11 @@ private extension String {
 }
 
 class NotificationTests: XCTestCase {
-    override func setUpWithError() throws {
+
+    @MainActor
+    override func setUp() async throws {
         setUpTestSuite()
-        WireMock.setUpScenario(scenario: "comment_flow")
+        try await WireMock.setUpScenario(scenario: "comment_flow")
```

Comment:
Notice this is not `async throws`, because it `try await`s on the async `WireMock.setUpScenario(scenario:)` method.